### PR TITLE
fix: fixing the loggers to handle both types of json outputs

### PR DESCRIPTION
**Pull request by millcityrunner**
Added in a parser helper for json output for loggers.

### CODE REVIEW

**Review by mbonig — APPROVED** [review]

File: src/handlers/index.py [hunk 1]
```diff
@@ -33,6 +34,23 @@ def handler(event, context):
...
+            for line in json_str.strip().split('\n'):
+                try:
+                    json_objects.append(json.loads(line))
+                except json.JSONDecodeError as e:
+                    logger.error(f"Failed to parse line: {line}, error: {e}")
+
+        except (TypeError, AttributeError) as e:
+            logger.error("Failed to parse json: %s" % e)
+
+        return json_objects
```
Comment (mikejgray) [inline src/handlers/index.py:53]: I'm guessing this function came about as a result of some inconsistent log data. Might be worth adding in some unit tests to handle each edge case as it arises and help guarantee regressions don't come in with future updates.

**Review by mikejgray — APPROVED** [review]

**Comment by millcityrunner** [comment]
```
andrew@andrews-MacBook-Pro kubectl-provider % python src/handlers/index_test.py
base case assert passed
multiple logs in one message assert passed
Failed to parse line: {"key": "value", error: Expecting ',' delimiter: line 1 column 16 (char 15)
incomplete log assert passed
more than 2 logs assert passed

```

**Review by mikejgray — APPROVED** [review]
